firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# Unreleased
+# Unreelased
+- [changed] Increases the aggressiveness of network retires when an app's
+  visibility status changes.
```

Comment:
Is "visibility" the right term here? Would it make more sense to just say "foreground status"?